src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -195,6 +196,7 @@ void testGetUsersApprovedForDataset() {
   void testInsertDraftDataAccessRequest() {
     User user = new User();
     user.setUserId(1);
+    user.setLibraryCards(List.of(new LibraryCard()));
```

Comment:
It was expected that these tests broke because the user needs a Library Card now.